iml-agent/src/cli.rs

```diff
@@ -283,13 +283,12 @@ pub enum App {
     /// Get latest kernel which supports listed modules
     GetKernel { modules: Vec<String> },
 
-    #[structopt(name = "try_mount")]
-    /// Try to mount `lustre_device` to the `mount_point`
-    TryMount {
-        #[structopt(long)]
+    #[structopt(name = "mount")]
+    /// Mount lustre device
+    Mount {
+        /// For example, 10.73.20.11@tcp0:10.73.20.12@tcp0:/fs2
```

Comment:
I wonder if this could be further improved by resolving a fs name to the the full spec, e. g. "fs2" to "10.73.20.11@tcp0:10.73.20.12@tcp0:/fs2". This would be more friendly.